addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -178,6 +181,7 @@
             <script type="text/javascript" src="/im_livechat/static/src/legacy/public_livechat.js"></script>
             <t t-call="web._assets_helpers"/>
             <link rel="stylesheet" type="text/scss" href="/im_livechat/static/src/scss/im_livechat_bootstrap.scss"/>
+            <link rel="stylesheet" type="text/scss" href="/im_livechat/static/src/legacy/public_livechat.scss"/>
```

Comment:
I don't think all of the variables (especially bootstrap) used in this file are defined in the bundle, we actually don't want bootstrap to be included as a dependency on external websites.

Reply:
13.3 had these styles: https://github.com/odoo/odoo/blob/saas-13.3/addons/im_livechat/views/im_livechat_channel_templates.xml#L189

They have been merged in a single file in 13.4 (`public_livechat.scss`), but was mistakenly omitted.
I prefer to play safe and keep stuffs as close to 13.3 as possible, until rewriting of public livechat with OWL.

Reply:
Ok, it seems to work at least